tests/test_decorators.py

```diff
@@ -0,0 +1,27 @@
+from tests import SiteTest
+
+class DecoratorTests(SiteTest):
+    def test_decorator_api_json(self):
+        """ Check the json validation decorator """
+        from pysite.decorators import api_params
+        from pysite.constants import ValidationTypes
+        from schema import Schema
+
+        SCHEMA = Schema([{"user_id": int, "role": int}])
+
+        @api_params(schema=SCHEMA, validation_type=ValidationTypes.json)
+        def try_json_type(data):
+            return data
+
+        try:
+            try_json_type("not json")
+        except Exception as error_message:
+            self.assertEqual(type(error_message), AttributeError)
```

Comment:
how about [`assertRaises`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaises) instead?
```py
with self.assertRaises(AttributeError):
    try_json_type("not json")
```

Reply:
I liked it, sorted :+1: